android/README.md

```diff
@@ -3,4 +3,15 @@
   See https://gradle.org/install#with-a-package-manager
 ## Build Weex   
 run `gradle build`   
-Playground app artifact will be under 'playground/app/build'
+Playground app artifact will be under 'playground/build'
+## The submodule weex-playground
+[Weex Playground](https://github.com/apache/incubator-weex-playground) can be used to test and preview weex pages on Android and it has been added as a submodule.
+### Relationship between weex and playground
+* The playground submodule will be cloned and updated automatically when developers clone the ‘incubator-weex’ repo and build it.In this case, playground project will implement the source code of weex-sdk.
+* When developer directly clone the 'weex-playground' repo and run it separately, the playground project will implement a corresponding aar version of weex-sdk.
+### When to publish SNAPSHOT
+When some commits of  playground rely on unpublished features of weex-sdk,a SNAPSHOT of weex-sdk will be needed to support it.
+### How to publish SNAPSHOT
+Details in `scripts/publish_snapshots.sh`   
```

Comment:
As `scripts/publish_snapshots.sh` only contains two lines, please delete that file and move its content to here